tool/util/execute.dart

```diff
@@ -47,13 +47,15 @@ Future<BetterProcessResult> exec(
       runInShell: runInShell
     );
 
+    var startMessage = "== Executing ${executable}"
+      " with arguments ${args} (pid: ${process.pid}) ==";
+
     if (raf != null) {
-      await raf.writeln(
-        "[${currentTimestamp}] == Executing ${executable}"
-          " with arguments ${args} (pid: ${process.pid}) =="
-      );
+      await raf.writeln("[${currentTimestamp}] ${startMessage}");
     }
 
+    dbg(startMessage);
```

Comment:
Adds debug messages when executing commands so that they can be correctly checked.